assembly/onpremises-ide-packaging-war-factory/src/main/java/com/codenvy/onpremises/factory/deploy/FactoryServletModule.java

```diff
@@ -39,6 +41,11 @@ protected void configureServlets() {
         filterRegex(PASS_RESOURCES_REGEXP).through(com.codenvy.onpremises.factory.filter.ReferrerCheckerFilter.class);
 
 
+        Multibinder<FactoryParametersResolver> githubFactoryParametersResolverMultibinder =
```

Comment:
also notice that the "We do not use servlet modules to bind components" rule is not written so it can't be required for PR